engine-primitives/engine-primitives/attributes.go

```diff
@@ -45,6 +46,10 @@ type PayloadAttributes struct {
 	// to the block currently being processed. This field was added for
 	// EIP-4788.
 	ParentBeaconBlockRoot common.Root `json:"parentBeaconBlockRoot"`
+
+	// PrevProposerPubKey carries the public key of previous block proposed
+	// Needed to enshrine distributeFor in EVM
+	PrevProposerPubKey crypto.BLSPubkey `json:"prevProposerPubKey"`
```

Comment:
```suggestion
	PrevProposerPubKey crypto.BLSPubkey `json:"parentProposerPubkey"`
```
nit: using this tag as it follows the pattern used by `parentBeaconBlockRoot`